test/gap-skipper.test.js

```diff
@@ -0,0 +1,81 @@
+import videojs from 'video.js';
+import QUnit from 'qunit';
+import {
+  useFakeEnvironment,
+  useFakeMediaSource,
+  createPlayer,
+  openMediaSource,
+  standardXHRResponse
+} from './test-helpers.js';
+
+QUnit.module('Adaptive Seeking', {
+  beforeEach() {
+    this.env = useFakeEnvironment();
+    this.requests = this.env.requests;
+    this.mse = useFakeMediaSource();
+    this.clock = this.env.clock;
+    this.old = {};
+
+    // setup a player
+    this.player = createPlayer();
+  },
+
+  afterEach() {
+    this.env.restore();
+    this.mse.restore();
+    this.player.dispose();
+  }
+});
+QUnit.test('Adaptive seeking skips over gap in firefox with waiting event', function() {
```

Comment:
remove `Adaptive seeking` from the test name since we are not using that name anymore